WombatProject/WombatLibrarianApi.csproj

```diff
@@ -5,6 +5,15 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Remove="Mappings\**" />
+    <Content Remove="Mappings\**" />
+    <EmbeddedResource Remove="Mappings\**" />
+    <None Remove="Mappings\**" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AutoMapper" Version="10.1.1" />
+    <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="8.1.1" />

```

Comment:
I removed unnecessary usings form all classes. I guess AutoMapper package wasn't used after all, if it is so, then it can be removed from the packages.

Reply:
Based on our conversation I removed the Automapper packages and I accept all changes :)